settings.py

```diff
@@ -0,0 +1,84 @@
+import ConfigParser
+import os.path as Path
+import os
+
+
+HOME_DIRECTORY = Path.expanduser('~')
+CONFIG_SECTION = 'Settings'
+CONFIG_FILENAME = 'resin.cfg'
+
+class Settings(object):
+
+	_setting = {
+		'pine_endpoint': 'https://api.resinstaging.io/ewa/',
+		'api_endpoint': 'https://api.resinstaging.io/',
+		'data_directory': Path.join(HOME_DIRECTORY, '.resin'),
+		# cache time : 1 week in milliseconds
+		'image_cache_time': (1 * 1000 * 60 * 60 * 24 * 7),
```

Comment:
@craig-mulligan, downloading image actually works. You can find it in models/os.py. Sorry, I've created the documentation for this. the python sdk has the same method as JS sdk which are mentioned here https://github.com/resin-io/resin-sdk/blob/master/DOCUMENTATION.md. Only few methods (2 or 3 as I reckon) are not fully implemented yet. Please ask me if you need any supports